lights/light_device/Lamp.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+
+namespace Lamp_Device{
+    class Lamp
+    {
+        private readonly Socket _client = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
+        private int PORT;
+
+        public Lamp(int port)
+        {
+            PORT = port;
+        }
+
+        public void StartConnection()
+        {
+            ConnectLoop();
+        }
+
+        public void StartSending()
+        {
+            SendLoop();
+        }
+
+        public void Close()
+        {
+            Exit();
+        }
+
+        private void Exit()
+        {
+            _client.Shutdown(SocketShutdown.Both);
+            _client.Close();
+            Environment.Exit(0);
+        }
+
+        private void SendLoop()
+        {
+            while (true)
+            {
+                Console.WriteLine("Enter a request: ");
+                string request = Console.ReadLine();
```

Comment:
nie używać inferencji typów np.
```c#
                var request = Console.ReadLine();
                var buffer = Encoding.ASCII.GetBytes(request);
                var receivedBytes = new byte[1024];
```
Moim zdaniem taki zapis jest troszkę którszy i czytelniejszy i ładniej się układa na stronie. Ale to tylko moja opinia